lib/msal-node/test/client/ClientCredentialClient.spec.ts

```diff
@@ -982,6 +982,88 @@ describe("ClientCredentialClient unit tests", () => {
         expect(authResult.state).toHaveLength(0);
     });
 
+    it("re-acquires a token via a network request, because the cached token's refreshOn value is expired", async () => {
```

Comment:
Test doesn't take into account ESTS being down, as per acceptance tests in the spec.